modules/dhp_connected_devices/lib/fitbit/client.rb

```diff
@@ -57,17 +57,32 @@ def get_auth_code(callback_params)
       # @return [nil]
       # @raise TokenRevocationError
       def revoke_token(token)
+        refresh_token = token[:refresh_token]
         resp = connection.post(config.revoke_token_base_path) do |req|
           req.headers = headers
-          req.body = "token=#{token[:refresh_token]}"
+          req.body = "token=#{refresh_token}"
         end
-        raise "response code: #{resp.status}, response body: #{resp.body}" unless resp.status == 200
+        raise "response code: #{resp.status}, response body: #{resp.body}" unless response_valid?(resp, refresh_token)
       rescue => e
         raise TokenRevocationError, e.message.to_s
       end
 
       private
 
+      def response_valid?(resp, refresh_token)
```

Comment:
Maybe this can be renamed to 
```suggestion
      def token_revoked?(resp, refresh_token)
```